packages/auto-bump/src/app.js

```diff
@@ -0,0 +1,29 @@
+import * as fs from "fs"
+import * as path from 'path'
+
+const updatePackageVersion = async (location, version) => {
+  const fileName = path.resolve(`packages/${location}`, 'package.json')
+  const workingPackageRaw = await fs.promises.readFile(fileName, {
+    encoding: 'utf-8',
+  })
+  if (!workingPackageRaw) {
+    spinner.fail('Working package.json not found. Please run in a javascript application')
+    process.exit(1)
+  }
+
+  const workingPackage = JSON.parse(workingPackageRaw)
+  workingPackage.version = version
+
+  await fs.promises.writeFile(fileName, `${JSON.stringify(workingPackage, null, 2)}\n`)
+}
+
+export default (app) => {
+  app.onAny(async (event) => {
```

Comment:
I don't like this bit, couldn't get `release.published` to work which was the event that should've been triggered (and was when debugging) just wouldn't call the function given to it 🤷‍♀️ so settled for onAny as the action handles the event for us